src/Resources/views/standard_layout.html.twig

```diff
@@ -113,7 +115,7 @@ file that was distributed with this source code.
     </head>
     <body
             {% block body_attributes -%}
-                class="sonata-bc skin-black fixed
+                class="sonata-bc {{ _skin }} fixed
```

Comment:
```suggestion
                class="sonata-bc {% block admin_lte_skin_class %}{{ _skin }}{% endblock %} fixed
```

I'm not against a skin option for simple usage, and a skin block to easily override the template when there is more complex needs (like a role dependant skin).